test/images/vulnerability-scan/scan.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+images=( "access-setup" "argocd-registrar" "cluster-setup" "gateway-deployment" "kcp-registrar" "shellcheck" )
+images_vul=("${images[@]/*/1}")
+
+get_digest() {
```

Comment:
Couldn't we just share the same function with what is used for single image scan?

Reply:
Hmm. It would require a lot of modification but yes, it could be made possible. 
But, even in the future when we write PipelineRuns for this, I was thinking we would have two different images under test /images - one for individual scan and one for periodic scan - meaning two Dockerfiles and respective scripts.

Reply:
Fine you can have multiple images that import some common scripts or functions, possibly in a separate file.

Reply:
Ack. Will work on this in a subsequent PR.